### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -24772,32 +24729,34 @@ msgstr "не вдалося знайти віддаленого помічник
 
 #: transport-helper.c:165 transport-helper.c:579
 msgid "can't dup helper output fd"
-msgstr ""
+msgstr "неможливо продублювати вивід помічника fd"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо зробити копію дескриптора файла виводу помічника"
```

File: po/uk.po [hunk 2]
```diff
@@ -24772,32 +24729,34 @@ msgstr "не вдалося знайти віддаленого помічник
...
 
 #: transport-helper.c:218
 #, c-format
 msgid ""
-"unknown mandatory capability %s; this remote helper probably needs newer version "
-"of Git"
+"unknown mandatory capability %s; this remote helper probably needs newer version of "
+"Git"
 msgstr ""
+"невідомий обовʼязковий параметр %s; цей віддалений помічник напевно потребує "
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"невідома обовʼязкова здібність %s; цей віддалений помічник напевно потребує "
```

File: po/uk.po [hunk 3]
```diff
@@ -24772,32 +24729,34 @@ msgstr "не вдалося знайти віддаленого помічник
...
+"новішої версії Git"
 
 #: transport-helper.c:224
 msgid "this remote helper should implement refspec capability"
-msgstr ""
+msgstr "цей віддалений помічник має реалізовувати можливість refspec"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "цей віддалений помічник має реалізовувати здібність refspec"
```

File: po/uk.po [hunk 4]
```diff
@@ -24811,85 +24770,85 @@ msgstr "не вдалося прочитати посилання %s"
...
 #: transport-helper.c:665 transport.c:1583
 msgid "operation not supported by protocol"
-msgstr ""
+msgstr "операція не підтримується протоколом"
 
 #: transport-helper.c:668
 #, c-format
 msgid "can't connect to subservice %s"
-msgstr ""
+msgstr "не вдається підключитися до підсервісу %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо підключитися до підсервісу %s"
```

File: po/uk.po [hunk 5]
```diff
@@ -24901,31 +24860,33 @@ msgid ""
 "No refs in common and none specified; doing nothing.\n"
 "Perhaps you should specify a branch.\n"
 msgstr ""
+"Немає спільних рефів і жодного не вказано; нічого не робиться.\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Немає спільних посилань і жодного не вказано; нічого не робиться.\n"
```

File: po/uk.po [hunk 6]
```diff
@@ -24901,31 +24860,33 @@ msgid ""
...
 #, c-format
 msgid "write(%s) failed"
-msgstr ""
+msgstr "write(%s) завершився невдало"
 
 #: transport-helper.c:1487
 #, c-format
 msgid "%s thread failed"
-msgstr ""
+msgstr "Потік %s завершився невдало"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "%s потік завершився невдало"
```

File: po/uk.po [hunk 7]
```diff
@@ -24935,26 +24896,26 @@ msgstr "%s потік не зміг приєднатись: %s"
 #: transport-helper.c:1510 transport-helper.c:1514
 #, c-format
 msgid "can't start thread for copying data: %s"
-msgstr ""
+msgstr "не вдається запустити потік для копіювання даних: %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо запустити потік для копіювання даних: %s"
```

File: po/uk.po [hunk 8]
```diff
@@ -24935,26 +24896,26 @@ msgstr "%s потік не зміг приєднатись: %s"
...
 
 #: transport-helper.c:1551
 #, c-format
 msgid "%s process failed to wait"
-msgstr ""
+msgstr "Процес %s не зміг дочекатися"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "%s процес не зміг дочекатися"
```

File: po/uk.po [hunk 9]
```diff
@@ -24935,26 +24896,26 @@ msgstr "%s потік не зміг приєднатись: %s"
...
 
 #: transport-helper.c:1555
 #, c-format
 msgid "%s process failed"
-msgstr ""
+msgstr "Процес %s завершився невдало"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "%s процес завершився невдало"
```

File: po/uk.po [hunk 10]
```diff
@@ -24935,26 +24896,26 @@ msgstr "%s потік не зміг приєднатись: %s"
...
 
 #: transport-helper.c:1573 transport-helper.c:1582
 msgid "can't start thread for copying data"
-msgstr ""
+msgstr "не вдається запустити потік для копіювання даних"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо запустити потік для копіювання даних"
```

File: po/uk.po [hunk 11]
```diff
@@ -24935,26 +24896,26 @@ msgstr "%s потік не зміг приєднатись: %s"
...
 
 #: transport.c:122
 #, c-format
 msgid "Would set upstream of '%s' to '%s' of '%s'\n"
-msgstr ""
+msgstr "Встановить першоджерельне сховище \"%s\" на \"%s\" від \"%s\"\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Встановить першоджерельне сховище для \"%s\" на \"%s\" в \"%s\"\n"
```

File: po/uk.po [hunk 12]
```diff
@@ -24964,48 +24925,51 @@ msgstr "не вдалося прочитати пакет \"%s\""
...
 
 #: transport.c:300
 msgid "see protocol.version in 'git help config' for more details"
 msgstr ""
+"Дивіться protocol.version у \"git help config\" для більш детальної інформації"
 
 #: transport.c:301
 msgid "server options require protocol version 2 or later"
-msgstr ""
+msgstr "опції сервера вимагають протокол версії 2 або вишче"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "опції сервера вимагають протокол версії 2 або пізнішої"
```

File: po/uk.po [hunk 13]
```diff
@@ -24964,48 +24925,51 @@ msgstr "не вдалося прочитати пакет \"%s\""
...
 
 #: transport.c:463
 msgid "server does not support wait-for-done"
-msgstr ""
+msgstr "сервер не підтримує wait-for-done"
 
 #: transport.c:817
 msgid "could not parse transport.color.* config"
-msgstr ""
+msgstr "не вдалося розібрати transport.color.* config"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "не вдалося розібрати конфігурацію transport.color.*"
```

File: po/uk.po [hunk 14]
```diff
@@ -24964,48 +24925,51 @@ msgstr "не вдалося прочитати пакет \"%s\""
...
 #, c-format
 msgid "unknown value for config '%s': %s"
-msgstr "невідоме значення для конфігураціі \"%s\": %s"
+msgstr "невідоме значення для конфігурації \"%s\": %s"
 
 #: transport.c:1092
 #, c-format
 msgid "transport '%s' not allowed"
-msgstr ""
+msgstr "транспортування \"%s\" не дозволено"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "засіб передачі \"%s\" не дозволений"
```

File: po/uk.po [hunk 15]
```diff
@@ -24964,48 +24925,51 @@ msgstr "не вдалося прочитати пакет \"%s\""
...
+msgstr "git-over-rsync більше не підтримується"
 
 #: transport.c:1249
 #, c-format
 msgid ""
 "The following submodule paths contain changes that can\n"
 "not be found on any remote:\n"
 msgstr ""
+"Наступні шляхи до підмодулів містять зміни, які неможливо\n"
+"знайти на жодному віддаленому посиланні:\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"знайти в жодному віддаленому призначенні:\n"
```

File: po/uk.po [hunk 16]
```diff
@@ -25022,38 +24986,48 @@ msgid ""
...
+"\n"
+"Будь ласка, спробуйте\n"
+"\n"
+"\tgit push --recurse-submodules=on-demand\n"
+"\n"
+"або перейдіть до директорії та використайте\n"
+"\n"
+"\tgit push\n"
+"\n"
+"щоб надіслати їх на віддалений сервер.\n"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"щоб надіслати їх до віддаленого призначення.\n"
```

File: po/uk.po [hunk 17]
```diff
@@ -25022,38 +24986,48 @@ msgid ""
...
 
 #: tree-walk.c:36
 msgid "too-short tree object"
-msgstr ""
+msgstr "занадто короткий обʼєкт дерева"
 
 #: tree-walk.c:42
 msgid "malformed mode in tree entry"
-msgstr ""
+msgstr "невірно сформований режим у записі дерева"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "невірно визначений режим у записі дерева"
```

**Review by arkid15r — COMMENTED** [review]
Great work!

File: po/uk.po (hunk 6 above)
Comment (kasya) [reply po/uk.po]: :+1:

**Pull request merged**